vispy/visuals/collections/agg_fast_path_collection.py

```diff
@@ -123,7 +123,7 @@ def append(self, P, closed=False, itemsize=None, **kwargs):
         """
 
         itemsize = itemsize or len(P)
-        itemcount = len(P) / itemsize
+        itemcount = int(len(P) / itemsize)
```

Comment:
why not just `//` here, too?

Reply:
Because `itemsize` could come from the user and could be a float. It's possible that `itemsize` being a float could break something else though. I'll check and make the changes if necessary.